README.md

```diff
@@ -136,6 +134,7 @@ If you already have an Angular application, you can follow the installation step
     	map: {
     	   ...
     	   '@clr/angular': 'node_modules/@clr/angular/bundles/clr-angular.umd.js',
+    	   '@clr/icons': 'node_modules/@clr/icons/bundles/clr-icons.umd.js',
```

Comment:
We have `clr-icons-all.min.js` instead of `clr-icons.umd.js`, right?

Reply:
This is systemjs, so we ship a UMD bundle separately from the other ones that are specific to what systemjs would use here.